metric-publishers/cloudwatch-metric-publisher/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+
+log4j.rootLogger=INFO, A1

```

Comment:
What are these ^M artifacts?

Reply:
No idea, I copied this file from another module. Maybe a different kind of line ending?

Reply:
Hopefully easy to 'fix'?